CONTRIBUTING.md

```diff
@@ -4,12 +4,13 @@
 
 The environment used for development and CI consists of:
 
-- a system installation of [`rustup`](https://rustup.rs/)
+- a system installation of [`rustup`](https://rustup.rs/) with the latest stable and nightly toolchains
 - a [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) environment containing all required Python packages
 
-Once `rustup` is installed, ensure that the latest stable toolchain is available by running
+Once `rustup` is installed, ensure that the latest stable and nightly toolchains are available by running
 
 ```
+rustup toolchain install nightly
```

Comment:
IIUC, this should install all components of the nightly Rust toolchain, though we should only need `rustfmt`. Is there a command we can use to _only_ install nightly `rustfmt`?

cc @andygrove

Reply:
```
rustup component add rustfmt --toolchain nightly
cargo +nightly fmt
```
Taken from https://github.com/rust-lang/rustfmt#on-the-nightly-toolchain